sccm/core/clients/manage/cmg/security-and-privacy-for-cloud-management-gateway.md

```diff
@@ -21,19 +21,19 @@ This article includes security and privacy information for the Configuration Man
 ## CMG security details
 - The CMG accepts and manages connections from CMG connection points. It uses mutual SSL authentication using certificates and connection IDs.
 - The CMG accepts and forwards client requests using the following methods:
-	- Pre-authenticates connections using mutual SSL with the PKI-based client authentication certificate or Azure AD. 
```

Comment:
Couldn't tell if the next 3 were supposed to be at the same level or if there are 2 children as shown in smart diff

Reply:
@nschonni 
Hey Nick, the 2 bullets starting with "IIS on the..." are subs/children of pre-auth bullet. The cert trust list bullet goes back to the same level as the pre-auth one.

Reply:
Fixed